### DISCUSSION TITLE
Copy URL → S3 URL creates useless S3 urls

### DISCUSSION THREAD

**Discussion by avilshadow** [iterate-ch/cyberduck#16550] [Q&A]
In some cases, when the path to an object contains.. for example symbol `=`, Cyberduck apply url encoding, and url becomes useless for `aws s3` terminal client.

For example, I get smt like this
s3://my-bucket/data/created%3D1731594551758/bucket%3D8/part-file-f07925cf0d0c.c000.snappy.parquet
should be
s3://my-bucket/data/created=1731594551758/bucket=8/part-file-f07925cf0d0c.c000.snappy.parquet

quick googling did not show that s3 urls should correspond to URI standard(I asked gemini, so I am not fully sure), but even if it should, encoded S3 URL points to nowhere for `aws s3` command line utility, i.e. it is completely useless.

**Comment by dkocher** [comment]
Unfortunately `aws s3 help` is not specific on URI encoding

> S3Uri: represents the location of a S3 object, prefix, or bucket.  This
       must  be  written in the form s3://mybucket/mykey where mybucket is the
       specified S3 bucket, mykey is the specified S3 key.  The path  argument
       must  begin with s3:// in order to denote that the path argument refers
       to a S3 object. Note that prefixes are separated  by  forward  slashes.
       For  example, if the S3 object myobject had the prefix myprefix, the S3
       key would be myprefix/myobject, and if the object was in the bucket my-
       bucket, the S3Uri would be s3://mybucket/myprefix/myobject.

but I can confirm the behaviour you describe.

**Reply by dkocher** [reply]
Opened #16551.
